src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/config/OpenApiConfiguration.kt

```diff
@@ -0,0 +1,55 @@
+package uk.gov.justice.digital.hmpps.personrecord.config
+
+import io.swagger.v3.oas.models.Components
+import io.swagger.v3.oas.models.OpenAPI
+import io.swagger.v3.oas.models.info.Contact
+import io.swagger.v3.oas.models.info.Info
+import io.swagger.v3.oas.models.security.SecurityRequirement
+import io.swagger.v3.oas.models.security.SecurityScheme
+import io.swagger.v3.oas.models.servers.Server
+import io.swagger.v3.oas.models.tags.Tag
+import org.springframework.boot.info.BuildProperties
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+
+@Configuration
+class OpenApiConfiguration(buildProperties: BuildProperties) {
+  private val version: String = buildProperties.version
+
+  @Bean
+  fun customOpenAPI(): OpenAPI = OpenAPI()
+    .servers(
+      listOf(
+        Server().url("https://hmpps-person-record-dev.hmpps.service.justice.gov.uk").description("Development"),
+        Server().url("https://hmpps-person-record.hmpps.service.justice.gov.uk").description("Pre-Production"),
+        Server().url("https://hmpps-person-record.hmpps.service.justice.gov.uk").description("Production"),
+        Server().url("http://localhost:8080").description("Local"),
+      ),
+    )
+    .tags(
+      listOf(
+        Tag().name("Search").description("APIs for person search"),
+        Tag().name("Jobs").description("Batch job endpoints"),
+      ),
+    )
+    .info(
+      Info().title("HMPPS Core Person Record")
+        .version(version)
+        .contact(Contact().name("HMPPS Digital Studio").email("feedback@digital.justice.gov.uk")),
```

Comment:
? do we have a team email address? Not a big deal, sure nobody actually uses this instead of Slack